pkg/capture/capturetypes/status.go

```diff
@@ -1,51 +1,75 @@
 package capturetypes
 
-// State enumerates the activity states of a capture
-type State byte
+import (
+	"fmt"
+	"sort"
+	"strings"
 
-const (
-	// StateInitializing means that the capture is setting up
-	StateInitializing State = iota + 1
-	// StateCapturing means that the capture is actively capturing packets
-	StateCapturing
-	// StateClosing means that the capture is fully terminating and it's held resources are
-	// cleaned up
-	StateClosing
-	// StateError means that the capture has hit the error threshold on the interface (set by ErrorThreshold)
-	StateError
+	"github.com/els0r/goProbe/pkg/types/hashmap"
 )
 
-func (cs State) String() string {
-	switch cs {
-	case StateInitializing:
-		return "initializing"
-	case StateCapturing:
-		return "capturing"
-	case StateClosing:
-		return "closing"
-	case StateError:
-		return "inError"
-	default:
-		return "unknown"
-	}
-}
+// State enumerates the activity states of a capture
```

Comment:
Just like with muscles in the body: if you don't use it, lose it.

Reply:
Already done while you were looking at the PR. Race condition, human-style :rofl: